### PR TITLE
Added BERT code sample to "readme" file.

### PR DESCRIPTION
**Pull request by kinarapa**
Added the BERT code sample to illustrate inferencing using OpenVINO™ intergration with Torch-ORT.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -212,6 +212,37 @@ If no provider options are specified by user, OpenVINO™ Execution Provider is
 backend = "CPU"
 precision = "FP32"
 ```
+
+## Code Sample
+
+Below is an example of how you can leverage OpenVINO™ integration with Torch-ORT in a simple NLP usecase. 
+A pretrained BERT model fine-tuned on the CoLA dataset from huggingface model hub is used to predict grammar correctness on a given input text.
```
Comment (saipj) [inline Readme.md]: Can we provide link for hugging face model for quick access?

File: Readme.md [hunk 2]
```diff
@@ -212,6 +212,37 @@ If no provider options are specified by user, OpenVINO™ Execution Provider is
 backend = "CPU"
 precision = "FP32"
 ```
+
+## Code Sample
```
Comment (saipj) [inline Readme.md:210]: Can we add Code Sample after Note. Note section was part of part of "Add ONNX Runtime for PyTorch to your PyTorch inference script" heading.

File: Readme.md (hunk 1 above)
Comment (kinarapa) [reply Readme.md]: Added

File: Readme.md (hunk 2 above)
Comment (kinarapa) [reply Readme.md:210]: Moved Code Sample after Note section.

**Pull request closed without merging**